typesafety/tests/test_sphinxautodoc.py

```diff
@@ -286,7 +258,7 @@ def function_with_annotated_varargs(an_int, *varargs: list, **varkwargs: dict):
     pass
 
 
-def function_with_complex_default_value(a_tuple: tuple=(1, 2, 3)):
+def function_with_complex_default_value(a_tuple: tuple = (1, 2, 3)):
```

Comment:
Does the standard require the spaces around the `=` sign when there's an annotation?

Reply:
Yes, otherwise we get a pylint error. And it's not even perfectly consistent, as annotations with parentheses around them need no spaces. *sigh*